pfcwd/main.py

```diff
@@ -3,9 +3,30 @@
 import click
 import swsssdk
 import os
+import sys
+
 from tabulate import tabulate
 from natsort import natsorted
 
+from sonic_py_common.multi_asic import get_external_ports
+from utilities_common import multi_asic as multi_asic_util
+from utilities_common import constants
```

Comment:
Please reorganize imports per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:

> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

And group alphabetically in each group. This should read:

```python
import os
import sys

import click
import swsssdk
from natsort import natsorted
from sonic_py_common.multi_asic import get_external_ports
from tabulate import tabulate
from utilities_common import multi_asic as multi_asic_util
from utilities_common import constants
```